cg/cli/workflow/tomte/base.py

```diff
@@ -0,0 +1,19 @@
+"""CLI support to create config and/or start TOMTE."""
```

Comment:
In Atlas, we have defined it as `Tomte`. Should we keep this naming consistent across our tools?https://atlas.scilifelab.se/infrastructure/tomte/tomte/

```suggestion
"""CLI support to create config and/or start Tomte."""
```

Reply:
Yes! I would change all my suggestions regarding changes into TOMTE for this instead to be consistent with Atlas